3IPA.md

```diff
@@ -0,0 +1,78 @@
+# 3IPA Overview (Draft)
+
+Daniel Masny ([@danielmasny](https://github.com/danielmasny))
+
+We describe the high level flow of IPA that uses a PRF, i.e. the Dodis-Yampolski PRF, to shard events by match key. This allows us to group user events efficiently. However it leaks the amount of user events. We can use DP to hide this leakage.
```

Comment:
A link to the paper for this would be useful

Reply:
https://eprint.iacr.org/2004/310